addons/website/static/src/js/website.backendDashboard.js

```diff
@@ -116,6 +113,10 @@ var Dashboard = Widget.extend(ControlPanelMixin, {
         });
     },
 
+    render_dashboards_header: function() {
+        this.$('.o_dashboard_common').replaceWith(QWeb.render('website.dashboard_header', {widget: this}));
```

Comment:
Wait ... what ? No. Here you replace the content by the content rendered once again. This seems very unnecessary and probably not efficient. Why did you did it ?